shared/src/utils/validators/is-radiogram-status.ts

```diff
@@ -0,0 +1,33 @@
+import { Type } from 'class-transformer';
+import { RadiogramAcceptedStatus } from '../../models/utils/radiogram-status/radiogram-accepted-status';
+import { RadiogramDoneStatus } from '../../models/utils/radiogram-status/radiogram-done-status';
+import type { RadiogramStatus } from '../../models/utils/radiogram-status/radiogram-status';
+import { RadiogramUnreadStatus } from '../../models/utils/radiogram-status/radiogram-unread-status';
+import { IsLiteralUnion } from './is-literal-union';
+
+class RadiogramStatusBase {
+    @IsLiteralUnion<RadiogramStatus['type']>({
+        unread: true,
+        accepted: true,
+        done: true,
+    })
+    public type: RadiogramStatus['type'];
+
+    constructor(type: RadiogramStatus['type']) {
+        this.type = type;
+    }
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const IsRadiogramStatus = () =>
```

Comment:
Prefer building `radiogramStatusTypeOptions` like in `exercise-simulation-event.ts`